### ISSUE TITLE
Screen reader testing

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1972]
Pages:
- [x] Global header
- [x] Global footer
- [x] Hero images
- [x] Left nav
- [x] Home
- [x] Getting Started Home
- [x] How Leasing Works
- [x] New Reporter Checklist
- [x] System Access
- [x] New Reporter Training
- [x] Reporting Home  _- Note: it did not read the alt text for the 3 images on this page. There is proper alt text in the CMS._
- [x] Oil & Gas Production
- [x] Oil & Gas Revenue
- [x] Solid Minerals
- [x] Geothermal
- [x] References (Home)
- [x] Statutes _- note: reading table well!_
- [x] FOGRMA Amendments _- note: I can't figure out how to just read the content under a new selected tab, it reads all the content before the tabs every time to select a different tab_
- [x] Rulemakings _- note: reading table well!_
- [x] Regulation Identifier Number 1012-AA29
- [x] Regulation Identifier Number 1012-AA27
- [ ] Regulation Identifier Number 1012-AA26
- [ ] Regulation Identifier Number 1012-AA25
- [ ] Regulation Identifier Number 1012-AA24
- [ ] Regulation Identifier Number 1012-AA23
- [ ] Regulation Identifier Number 1012-AA22
- [ ] Regulation Identifier Number 1012-AA21
- [ ] Regulation Identifier Number 1012-AA20
- [ ] Regulation Identifier Number 1012-AA17
- [ ] Regulation Identifier Number 1012-AA15
- [ ] Regulation Identifier Number 1012-AA14
- [ ] Regulation Identifier Number 1012-AA13
- [ ] Regulation Identifier Number 1012-AA11
- [ ] Regulation Identifier Number 1010-AD70
- [ ] Regulation Identifier Number 1012-AA08
- [ ] Regulation Identifier Number 1012-AA05
- [ ] Regulation Identifier Number 1012-AA04
- [ ] Regulation Identifier Number 1012-AA03
- [ ] Regulation Identifier Number 1012-AA02
- [ ] Regulation Identifier Number 1010-AD32
- [ ] Regulation Identifier Number 1010-AD28
- [ ] Regulation Identifier Number 1010-AD22
- [ ] Regulation Identifier Number 1010-AD20
- [ ] Regulation Identifier Number 1010-AD05
- [ ] Regulation Identifier Number 1010-AD04
- [ ] Regulation Identifier Number 1010-AD01
- [ ] Regulation Identifier Number 1010-AD00
- [ ] Regulation Identifier Number 1010-AC86
- [ ] Regulation Identifier Number 1010-AC72
- [ ] Regulation Identifier Number 1010-AC70
- [ ] Regulation Identifier Number 1010-AC59
- [ ] Regulation Identifier Number 1010-AC51
- [ ] Regulation Identifier Number 1010-AC40
- [ ] Regulation Identifier Number 1010-AC38
- [ ] Regulation Identifier Number 1010-AC30
- [ ] Regulation Identifier Number 1010-AC25
- [ ] Regulation Identifier Number 1010-AC21
- [ ] Regulation Identifier Number 1010-AC08
- [ ] Regulation Identifier Number 1010-AC06
- [ ] Regulation Identifier Number 1010-AC00
- [ ] Regulation Identifier Number 1010-AB57
- [ ] Regulation Identifier Number 1010-AB17
- [ ] 1989 Coal Rule: Revision of Coal Product Valuation Regulations and Related Topics
- [x] Federal Register Notices
- [x] Information Collection Requests
- [x] OMB Control Number 1012-0001
- [ ] OMB Control Number 1012-0002
- [ ] OMB Control Number 1012-0003
- [ ] OMB Control Number 1012-0004
- [ ] OMB Control Number 1012-0005
- [ ] OMB Control Number 1012-0006
- [ ] OMB Control Number 1012-0008
- [ ] OMB Control Number 1012-0009
- [ ] OMB Control Number 1012-0010
- [x] OMB Control Number 1012-0012
- [ ] OMB Control Number 1012-0007
- [ ] OMB Control Number 1010-0090
- [ ] OMB Control Number 1010-0140
- [ ] OMB Control Number 1010-0169
- [ ] OMB Control Number 1010-0110
- [ ] OMB Control Number 1010-0074
- [ ] OMB Control Number 1010-0088
- [ ] OMB Control Number 1010-0138
- [ ] OMB Control Number 1010-0104
- [ ] OMB Control Number 1010-0095
- [ ] OMB Control Number 1010-0075
- [ ] OMB Control Number 1010-0061
- [ ] OMB Control Number 1010-0113
- [ ] OMB Control Number 1010-0135
- [ ] OMB Control Number 1010-0129
- [ ] OMB Control Number 1010-0126
- [ ] OMB Control Number 1010-0042
- [ ] OMB Control Number 1010-0121
- [ ] OMB Control Number 1010-0123
- [x] Forms _- note: I'm having issues with NVDA not automatically reading content under a new selected tab. The "insert" button is supposed to be used with the down arrow to start reading a page and when I do that it initiates the scroll instead._
- [x] Reference Lists and Codes
- [x] Handbooks
- [x] Minerals Revenue Reporter Handbook
- [x] Minerals Production Reporter Handbook
- [x] Solid Minerals Reporter Handbook
- [x] Geothermal Payor Handbook - Class 1 Leases
- [x] Geothermal Payor Handbook - Class 2 & 3  Leases
- [x] Indian Payor Handbook
- [x] Valuation and Pricing
      _- Note: in "How Valuation Works" the screen reader didn't pick up the RVPA image alt text, but it picked up the alt text for the RVLA image. Both images have good alt text in the CMS. Not sure why it's reading out one but not the other._
- [x] Interactive Example (2016 Valuation Rule Federal Processed Gas Reporting Example)
     _- Note: the yellow caution and purple info icons need alt text, right now it's saying "ONRR image" and that's it. It's reading all the alt text for the equations I checked though! (Didn't check every equation)._
- [x] Interactive Example (Percent-of-Proceeds (POP) Contract Reporting Pre-2016) _Note: the screen reader didn't pick up the alt text for the Product Codes image. It picked up the alt text for all the other images._
- [x] Reporter Letters
- [x] Reporter Training
- [x] Paying (Home)
- [x] Payment Options
- [x] Late Payment Interest   _- Note: it did not read the alt text for the interest calculator formula image.. it just said "ONRR image". There is proper alt text in the CMS, not sure why it's picking up some image's alt text and not others._
- [x] Unpaid Debt
- [x] Compliance & Enforcement (Home)
- [x] Compliance  _- Note: it did not read the alt text for the "Data Mining Review" image. There is proper alt text in the CMS, and it picked up the other images alt text._
- [x] Enforcement & Litigation Support
- [x] Appeals
- [x] Cooperative Agreements (STRAC)
- [x] Indian Resources (Home)
- [x] Assistance _- Note: it did not read the alt text for the "IESE Logo" image. There is proper alt text in the CMS, and it picked up the other images alt text._
- [x] Tribal Consultation
- [x] Indian Pricing _- Note: it did not read the alt text for the 3 gas maps. There is proper alt text in the CMS, and it picked up the oil map alt text._
- [x] Indian References
- [x] Tribal Lockbox Payments
- [x] About ONRR
- [x] Public Affairs _- Note: it did really well reading the press releases!_
- [x] Organization & Leadership
- [x] Office Locations
- [x] Employment _- Note: it did not read the "Healthy Workplace" and "USA Jobs" logos. There is proper alt text in the CMS._
- [x] Engagement
- [x] Indian Oil Valuation Negotiated Rulemaking Committee
- [x] Interagency Coordination
- [x] Freedom of Information Act
- [x] Anti-Harassment Program
- [x] Contact Us
- [ ] Search Results
- [x] Getting Started
- [x] System Access
- [x] Leases & Agreements
- [ ] Oil & Gas Production
- [ ] Oil & Gas Royalty
- [ ] Oil & Gas Rent
- [ ] Solid Minerals
- [ ] Geothermal
- [ ] Payments & Debt Collection
- [ ] Updating Contact Info
- [ ] Valuation & Pricing
- [x] Compliance & Enforcement
- [x] Indian Services
- [ ] Contacts by Company
- [ ] Information Requests
- [x] Events
- [ ] Glossary
- [x] Sitemap
- [x] Disclaimer

**Comment by cthomasONRR** [comment]
@mcharg I've checked all pages except for the contacts pages. I'll do that tomorrow (need a break from NVDA). 

I only check 2 of the rulemakings and ICR pages each, let me know if you want me to go back and check all of those. They are simple with just text and sometimes a bulleted list, so I didn't feel the need to check them all.

**Comment by mcharg** [comment]
@cthomasONRR Can you create an issue for all the images that aren't reading alt text?

I don't think it's necessary to go through all the rulemaking and ICR pages.

**Comment by cthomasONRR** [comment]
@mcharg OK will do. I'll check the contacts pages this afternoon (just can't get myself to use NVDA right now lol)

**Comment by cthomasONRR** [comment]
@mcharg For all contacts pages so far, it's picking up heading level 2:
![image](https://user-images.githubusercontent.com/72526208/172624051-6ca023c1-0681-4cb5-ba44-92a2489166f3.png)

![image](https://user-images.githubusercontent.com/72526208/172623993-b7ea26d7-99af-4783-9458-ceaa1e9e1200.png)

**Comment by mcharg** [comment]
@cthomasONRR We should create an issue for that, since on some pages it needs to be H2, but if there's already a tab title that's H2, they should be H3.

**Comment by cthomasONRR** [comment]
@mcharg I think we can close this issue. We have new issues addressing the screen reader shortcomings.
